pytest/lib/mocknet.py

```diff
@@ -83,7 +85,7 @@ def list_validators(node):
 
 def setup_python_environment(node, wasm_contract):
     m = node.machine
-    print(f'INFO: Setting up python environment on {m.name}')
+    logger.info(f'INFO: Setting up python environment on {m.name}')
```

Comment:
I guess `INFO` can be removed since logger.info will tell?

Reply:
Currently, I don't have the logger formatted to show the what the log level is at. I can added it if necessary, but would also end up being added to every other log as well that didn't have `INFO:` prefixed before